lib/data/contributor_data.dart

```diff
@@ -8,5 +8,12 @@ List<Contributors> contributorsDataList = [
       bio: "Tech Enthusiast | Self Learner | Android, Web, Desktop Developer.",
       profileUrl: "https://apilpoudel.com.np",
       chipDataString: ["C", "C++", "Flutter", "Firebase", "Digital Marketing"]),
-//You can add your data here to show up as a contributor.
```

Comment:
Please do not remove comments on the code. Instead, you can move the comment at the end of your data.